content-src/components/TopSites/TopSites.js

```diff
@@ -43,7 +44,7 @@ const TopSites = React.createClass({
           return (<div key={site.lastVisitDate || i}>
             <a onClick={() => this.onClick(i)} className={classNames("tile", {active: isActive})} href={site.url}>
               <SiteIcon className="tile-img-container" site={site} faviconSize={32} showTitle />
-              <div className="tile-close-icon" onClick={(ev) => {
+              <div hidden={site.type === FIRST_RUN_TYPE} className="tile-close-icon" onClick={(ev) => {
```

Comment:
Unfortunately we can't use that strategy because refs don't really work on repeated elements. We need to refactor TopSites to have a child component